packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -829,6 +856,21 @@ const Transaction = memo(function Transaction(props) {
         ...(_unmatched && { opacity: 0.5 }),
       }}
     >
+      {splitError && listContainerRef.current && (
+        <Popover
+          triggerRef={triggerRef}
+          isOpen
+          isNonModal
+          style={{ width: 375, padding: 5, maxHeight: '38px !important' }}
+          shouldFlip={false}
+          placement="bottom end"
+          arrowSize={updateId}
```

Comment:
Would using a key based on the allTransactions also solve the issue?

Reply:
Good idea about using `key`! That works too!

Reply:
Ok, so actually this won't work. Unit tests caught an issue with react re-renders. Basically by setting `key` we are implicitly memoizing the split error content. So in some circumstances the root component might get re-rendered (causing `updateId` to reset back to `1`). Which would then return the old split error instead of rendering a new one.

Reply:
I think it should work if we base the key on the `splitError` or the error amount. It will rerender everytime the split error amount changes or if the no longer any remaining splits.

Reply:
Yea, I tried that too. Unfortunately it has to be based on the combination of `allTransactions` + `splitError`.

`allTransactions` contains schedules which sometimes load with a small delay (meaning the page content gets pushed down so we need to recalculate the popover positioning).